tutorials/quickstart/tutorial.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: "NServiceBus Quick Start"
-reviewed: 2022-06-28
-isLearningPath: true
```

Comment:
@sergioc Can you give some context for why you made this change? @tamararivera and I are reviewing the Quickstart now as part of her onboarding and we noticed that the link to "the learning path" shows up on Part 2 and Part 3 of the tutorial, but is missing from Part 1 and it seems a bit weird. We would expect either for it to show up on all the pages, none of the pages, or just Part 1, but not the current iteration.

Reply:
Context: https://github.com/Particular/DeveloperEducation/blob/master/rfc-archive/2023-12-improve-nsb-marketing.md#docs-pages-miscellaneous-updates

Thanks for catching this! [Fixed](https://github.com/Particular/docs.particular.net/pull/6576).